mindbridge-ai/angularjs-slider
# Wc 3758 decimal in monetary value facet

This PR adds decimal support to rzslider as well as an equally crazy form of range validation to what was there previously.
It also adds this method to the slider template's scope for setting the step attribute in all of the inputs:
```js
getInputStepAttribute: function() {
    return 1 / Math.pow(10, this.slider.precision);
}
```

https://squanto.atlassian.net/browse/WC-3758

Steps to test or reproduce issue:
1. Copy the `/dist/rzslider.js` from this repo to  /squanto-web-audit/node_modules/angularjs-slider/dist/rzslider.js
2. Restart Gulp
3. Go to data table for AR so monetary value facet appears.
4. Try entering decimals in textboxes which are embedded into monetary value slider.
5. Refresh. Verify decimals remain in address bar.
